src/main/java/org/mockito/internal/verification/VerificationDataImpl.java

```diff
@@ -5,22 +5,22 @@
 package org.mockito.internal.verification;
 
 import org.mockito.internal.invocation.InvocationMatcher;
-import org.mockito.internal.stubbing.InvocationContainer;
-import org.mockito.invocation.MatchableInvocation;
+import org.mockito.internal.stubbing.InvocationContainerImpl;
 import org.mockito.internal.verification.api.VerificationData;
 import org.mockito.invocation.Invocation;
+import org.mockito.invocation.MatchableInvocation;
+
+import java.util.List;
 
 import static org.mockito.internal.exceptions.Reporter.cannotVerifyToString;
 import static org.mockito.internal.util.ObjectMethodsGuru.isToStringMethod;
 
-import java.util.List;
-
 public class VerificationDataImpl implements VerificationData {
 
     private final InvocationMatcher wanted;
-    private final InvocationContainer invocations;
+    private final InvocationContainerImpl invocations;
```

Comment:
I dont think this is necessary?

Reply:
InvocationContainer is a marker interface at this stage to hide the internal implementation from leaking to the public API. It's impractical to expose it as public API. Hence, we use the 'impl' everywhere in the codebase. For more info, take a look at the javadoc for InvocationContainer interface.

Hope that helps!